build eagle_default

help please!

Im am using master branch here and try to build eagle_default firmware but i get stuck in:

Adding DF driver: isl29501
CMake Error at CMakeLists.txt:344 (add_subdirectory):
  add_subdirectory given source "src/drivers/uart_esc" which is not an
  existing directory.

--- a/cmake/configs/qurt_sdflight_default.cmake
+++ b/cmake/configs/qurt_sdflight_default.cmake
@@ -60,7 +60,7 @@ set(config_module_list
 	# PX4 drivers
 	#
 	drivers/gps
-	drivers/uart_esc
+	drivers/pwm_out_rc_in
 	drivers/qshell/qurt
 
 	#

--- a/src/modules/systemlib/print_load_posix.c
+++ b/src/modules/systemlib/print_load_posix.c
@@ -54,6 +54,11 @@
 #include <mach/mach.h>
 #endif
 
+#ifdef __PX4_QURT
+// dprintf is not available on QURT. Use the usual output to mini-dm.
+#define dprintf(_fd, _text, ...) ((_fd) == 1 ? PX4_INFO((_text), ##__VA_ARGS__) : (void)(_fd))
+#endif
+
 extern struct system_load_s system_load;
 
 #define CL "\033[K" // clear line

